diff --git a/./imgui_git/backends/imgui_impl_opengl2.cpp b/./imgui/backends/imgui_impl_opengl2.cpp
index 0028369..2a2a978 100644
--- a/./imgui_git/backends/imgui_impl_opengl2.cpp
+++ b/./imgui/backends/imgui_impl_opengl2.cpp
@@ -1,3 +1,8 @@
+// Warning!
+// This file has been modified for ofxImGui to support GLES1.
+// This is not the original one. Changes are indicated.
+//----
+
 // dear imgui: Renderer Backend for OpenGL2 (legacy OpenGL, fixed pipeline)
 // This needs to be used along with a Platform Backend (e.g. GLFW, SDL, Win32, custom..)
 
@@ -55,6 +60,12 @@
 #pragma clang diagnostic ignored "-Wnonportable-system-include-path"
 #endif
 
+// --- BEGIN CUSTOM MODIFICATION
+#include "ofxImGuiConstants.h"
+#if defined(OFXIMGUI_RENDERER_GLES)
+#include "gles1CompatibilityHacks.h"
+#else// --- END CUSTOM MODIFICATION
+
 // Include OpenGL header (without an OpenGL loader) requires a bit of fiddling
 #if defined(_WIN32) && !defined(APIENTRY)
 #define APIENTRY __stdcall                  // It is customary to use APIENTRY for OpenGL function pointer declarations on all platforms.  Additionally, the Windows OpenGL header needs APIENTRY.
@@ -68,6 +79,7 @@
 #else
 #include <GL/gl.h>
 #endif
+#endif // CUSTOM OFXIMGUI ADDED LINE
 
 struct ImGui_ImplOpenGL2_Data
 {
